docs/advanced/Task-Reminders.md

```diff
@@ -0,0 +1,25 @@
+---
+publish: false
+---
+
+# Notifications
+
+<span class="related-pages">#plugin/reminder</span>
+
+Within Tasks, reminder notifications can be set using the standard Tasks format `⏲️ YYYY-MM-DD` for daily notifications at a set or by specifying the hour `⏲️ YYYY-MM-DD h:mm a`. Multiple reminders can be set by separating secondary reminders with a comma: `⏲️ 2000-03-24, 2000-03-28 10:05 am, 2000-03-31`
+
+## Limitnations
+
+- It's not posible to set a reminder for midnight 12:00 am. This due to a limination with how tasks creates dates using momentjs which sets the defaults time to midnight when one isn't provided.
+- System notifications don't work on Mobile because Obsidian doesn't provide an API
+
+## How to complete the reminder
+
+The reminder date doesn't change when completing the task, the date will change only when you complete it from the reminder popup or from the notification.
+
+![image](https://user-images.githubusercontent.com/38974541/143463881-e4af4b91-426f-48e8-938e-4a1053b06677.png)
+![image](https://user-images.githubusercontent.com/38974541/143464983-542675ae-a467-41c0-aaca-1075c42f8328.png)
+
+## Acknowledgment
+
+This feature was created using code from uphy's [obsidian-reminder](https://github.com/uphy/obsidian-reminder) plugin.
```

Comment:
What are the licensing conditions of that code?

Is there somewhere formal that we required to add credit to it?

Reply:
[MIT License](https://github.com/uphy/obsidian-reminder/blob/master/LICENSE) 

No expert, would that just mean including `Copyright (c) 2021 Liam Cain` in the source files that were taken directly? i.e modal + Notifications.ts

Reply:
Thank you. I suspect that is the wrong name in the copyright.
Have opened an issue on the project to check: https://github.com/uphy/obsidian-reminder/issues/149

Reply:
Resolving to make this PR easier for me to navigate. Added this as a follow-up task on my project board https://github.com/users/Erik-Handeland/projects/2/views/1

Reply:
> Resolving to make this PR easier for me to navigate. Added this as a follow-up task on my project board https://github.com/users/Erik-Handeland/projects/2/views/1

OK, thanks for the link. I've saved it.

Please let me know when you are ready for me to review what you have done so far.